io/src/main/java/org/commonjava/maven/ext/io/rest/DefaultTranslator.java

```diff
@@ -101,15 +105,21 @@
      * @param endpointUrl is the URL to talk to.
      * @param restMaxSize initial (maximum) size of the rest call; if zero will send everything.
      * @param restMinSize minimum size for the call
-     * @param repositoryGroup the group to pass to the endpoint.
+     * @param repositoryGroup the group to pass to the endpoint, still can be used, but is replaced by
+     *        brewPullActive flag and mode when used in combination with DA 2.1 and above
+     * @param brewPullActive flag saying if brew pull should be used for version retrieval
```

Comment:
So just to clarify, `repositoryGroup` isn't deprecated as such - as its still in use, until the next PNC release when it can be removed? (i.e. we'll need a separate ticket to remind us to remove the flag for that?)

Reply:
Yeah, we introduced `brewPullActive` and `mode` which tells DA which repository group to use (if any). The `repositoryGroup` parameter is meant to be used in combination with DA 2.0 and with DA 2.1 we will switch to use the other 2.

Reply:
Ok thanks! Can another ticket be spawned to note removal of `repositoryGroup` please?

Reply:
Created as NCL-6528. Thank you!